person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] Please implement getters and setters for the properties of the Person class, as per [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=Getters%20for%20%40id%2C%20%40name,name%20and%20%40age.) requirement.